third_party/gpus/rocm_configure.bzl

```diff
@@ -314,7 +317,7 @@ def _select_rocm_lib_paths(repository_ctx, libs_paths, bash_bin):
 
     return libs
 
-def _find_libs(repository_ctx, rocm_config, hipfft_or_rocfft, bash_bin):
+def _find_libs(repository_ctx, rocm_config, hipfft_or_rocfft, bash_bin, enable_dcu):
```

Comment:
it's strongly recommended to make `enable_dcu` a boolean instead of a string. If `'1'` stands for `true`, how about `'true'`, `'yes'`, `'on'` ?

Reply:
Have change enable_dcu to boolean.